### ISSUE TITLE
Querying with skip doesn't work

### ISSUE THREAD

**Issue by rvlewerissa** [prisma/prisma#2972]
**Describe the bug**
Querying with skip param doesn't work.

**To Reproduce**
1. Setup new project (prisma init <project-name>)
2. Deploy to demo server
3. Add bunch of users
```
mutation {
  createUser(data: {
    name: "name1"
  }) {
   name 
  }
}

mutation {
  createUser(data: {
    name: "name2"
  }) {
   name 
  }
}

```
4. Query with skip params
```
query {
  users(where: {}, skip: 1) {
    name
  }
}
```

**Expected behavior**
User will be skipped, only one user is showing

**Screenshots**
<img width="1440" alt="screen shot 2018-08-29 at 3 40 44 pm" src="https://user-images.githubusercontent.com/15160023/44776274-ee3f2480-aba1-11e8-8e7d-dafb4e37a290.png">

**Versions (please complete the following information):**
 - OS: `OS X High Sierra`
 - Prisma CLI: `prisma/1.14.2 (darwin-x64) node-v8.11.3`
 - Prisma Server: Demo Server

**Comment by divyenduz** [comment]
I am observing this issue for Prisma server `prismagraphql/prisma:1.12` as well.

**Comment by mcmar** [comment]
If you're able to change your code, I recommend querying with `after`, which I'm using and I can verify does work.

**Comment by mavilein** [comment]
Thanks for reporting and providing thorough descriptions! 🙏  We fixed this issue and it is available in version `1.15.2` 🎉 . The build will be available in a few minutes.
